backend/controller/dal/dal.go

```diff
@@ -794,16 +807,17 @@ func (d *DAL) loadDeployment(ctx context.Context, deployment dalsql.GetDeploymen
 		Key:      deployment.Deployment.Key,
 		Schema:   deployment.Deployment.Schema,
 	}
-	artefacts, err := d.db.GetDeploymentArtefacts(ctx, deployment.Deployment.ID)
+	darts, err := d.db.GetDeploymentArtefacts(ctx, deployment.Deployment.ID)
```

Comment:
Shouldn't this be calling into the artefacts registry instead?

Reply:
That was considered. It draws deployments into the domain. Since deployments depend on this association but artifacts don't I left this as is.

Reply:
I mentioned this in Slack, but the artefact repository will need to contain the mapping of deployment key to []artefacts, and the schema - basically everything needed to materialise a deployment on separate clusters.

Reply:
moved this operation into the registry abstraction

Reply:
in a successive change deployments will also be moved into a registry